src/cascadia/LocalTests_TerminalApp/TestHostApp/TestHostApp.vcxproj

```diff
@@ -0,0 +1,148 @@
+      <Project>{CA5CAD1A-C46D-4588-B1C0-40F31AE9100B}</Project>
+    </ProjectReference>
+    <ProjectReference Include="$(OpenConsoleDir)src\cascadia\TerminalSettings\TerminalSettings.vcxproj" />
+    <ProjectReference Include="$(OpenConsoleDir)src\cascadia\TerminalControl\TerminalControl.vcxproj" />
+
+    <ProjectReference Include="$(OpenConsoleDir)src\cascadia\TerminalApp\TerminalApp.vcxproj">
+      <Project>{ca5cad1a-44bd-4ac7-ac72-f16e576fdd12}</Project>
+    </ProjectReference>
+
+  </ItemGroup>
+
+  <PropertyGroup>
+    <!-- Some helper paths to find our test code -->
+    <_TestBinRoot>$(OpenConsoleDir)\bin\$(Platform)\$(Configuration)</_TestBinRoot>
+    <_CppWinrtBinRoot>$(OpenConsoleDir)\$(Platform)\$(Configuration)</_CppWinrtBinRoot>
+    <_CppWinrtBinRoot Condition="'$(Platform)'=='Win32'">$(OpenConsoleDir)\$(Configuration)</_CppWinrtBinRoot>
+    <_TAEFPlatformName>$(Platform)</_TAEFPlatformName>
+    <_TAEFPlatformName Condition="'$(Platform)'=='Win32'">x86</_TAEFPlatformName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.VisualStudio.TestPlatform.TestExecutor.WinRTCore">
+      <HintPath>$(OpenConsoleDir)\packages\Taef.Redist.Wlk.10.48.200103003-develop\lib\Microsoft.VisualStudio.TestPlatform.TestExecutor.WinRTCore.winmd</HintPath>
+      <IsWinMDFile>true</IsWinMDFile>
+
+      <!-- This path is _relative to the .winmd_ -->
+      <Implementation>..\build\Binaries\$(_TAEFPlatformName)\TE.AppxUnitTestClient.dll</Implementation>
+    </Reference>
+  </ItemGroup>
+
+  <Import Project="$(OpenConsoleDir)\packages\Microsoft.UI.Xaml.2.3.191217003-prerelease\build\native\Microsoft.UI.Xaml.targets" Condition="Exists('$(OpenConsoleDir)\packages\Microsoft.UI.Xaml.2.3.191217003-prerelease\build\native\Microsoft.UI.Xaml.targets')" />
+
+  <!-- Use this to auto-find all the dll's that TerminalConnection produces. We
+    don't roll these up automatically, so we'll need to copy them manually
+    (below) -->
+  <ItemGroup>
+      <TerminalConnectionDlls Include="$(_CppWinrtBinRoot)\TerminalConnection\*.dll"/>
+  </ItemGroup>
+
+  <Import Project="$(OpenConsoleDir)\src\common.build.post.props" />
+
+  <Target Name="AfterBuild">
+
+    <!-- Copy the AppxManifest.xml to another file, because when TAEF is
+      deploying the app, it'll delete the AppxManifest.xml file from this
+      directory when it tries to clean up after itself. -->
+    <Copy SourceFiles="$(TargetDir)\AppxManifest.xml" DestinationFiles="$(TargetDir)\TestHostAppXManifest.xml" />
+
+    <!-- Copy our test code from LocalTests_TerminalApp into this directory -->
+    <Copy SourceFiles="$(_TestBinRoot)\LocalTests_TerminalApp\TerminalApp.LocalTests.dll"
+          DestinationFiles="$(TargetDir)\TerminalApp.LocalTests.dll" />
+
+    <!-- Copy some dlls which TerminalConnection is dependent upon that didn't
+      get rolled up into this  directory -->
+    <Copy SourceFiles="@(TerminalConnectionDlls)"

```

Comment:
So, here's the deal. The dependencies from TerminalConnection get rolled up in the `GetPackagingOutputs` step, when it produces the "appx recipe". This means they only show up when the build produces an `AppX\` folder for either running or packaging.

It is literally impossible to produce an `AppX\` folder using MSBuild without packaging an appx, and we don't want to do that anyway, so the copy rules get to stay.

Might be worth doccing.